packages/webapp/public/locales/en/translation.json

```
@@ -156,6 +156,8 @@
     "WILD_CROP": "Wild crops"
   },
   "ANIMAL": {
+    "ADD_MORE_BODY_TEXT": "Do you own other animal type or a different breed?",
```

```suggestion
    "ADD_MORE_BODY_TEXT": "Do you own another animal type or a different breed?",
```

Oh dear, there are two different texts for this depending on where in Figma you look 😂 Also neither is perfectly grammatical. I'll ping Loïc to see if he wants "own" or "have" here.